src/administration/users.md

```diff
@@ -39,26 +39,22 @@ If you want your users to be able to see everything (Reader), but only commit to
 
 ## Manage user permissions at the project level
 
-Access the project-level configuration.
+From your list of projects, enter into the project where you want to view or edit user permissions. At this point, you will not have entered a particular environment. Click the Settings tab at the top of the page, the click the ‘Access’ tab on the left to show the project-level users and their roles. 
 
-![Project configure icon](/images/project_w-configarrow.png)
+![Project user management screenshot](/images/settings_project_access.png)
 
-Click the project configuration icon next to the project name in the upper-left corner of the project page.
-
-------------------------------------------------------------------------
-
-Manage users.
-
-![Project user management screenshot](/images/project_usermanagement.png)
-
-The `Access control` tab shows project-level users and their roles.
+The `Access` tab shows project-level users and their roles.
 
 Selecting a user will allow you either to edit that user's permissions or delete the user's access to the project entirely.
 
-Add a new user by clicking on the `Add user` button.
+Add a new user by clicking on the `Add` button.
 
 If you select the 'Reader' role for the user, you'll have the option of adjusting the user's permissions at the environment level.
 
+From this view, you can assign the user’s access. Selecting them to become a ‘Project admin’ will give them ‘Admin’ access to every environment in the project. Alternatively, you can give the user ‘Admin’, ‘Viewer’, ‘Contributor’, or ‘No Access’ to each environment separately. 
+
+If you select the ‘Viewer’ role for the user, you’ll have the option of adjusting the user’s permissions at the environment level. 
```

Comment:
The new lines here use "Viewer".  The existing line above uses "Reader".  Which is the new UI using?  It should be consistent.

Reply:
Also these appear to be using typographic single quotes.  That's... not a thing.  Don't use that.  Use backticks to get code style formatting, use non-typographic double-quotes for normal English quotations.  Typographic quotes are the devil.  (Very very common source of character encoding mismatches and horrible data integrity problems.  I have scars from the number of hours I've wasted fighting with them.)

That applies to the apostrophe here, too.

Reply:
Are you unable to adjust per-env permissions if you select Contributor?  That's not clear.

Reply:
It's Viewer - so I'll change that. I'll update to "Viewer" as well.